tests/units/test_units.py

```diff
@@ -176,6 +176,7 @@ def test_added_degrees_units():
     assert units('degrees_north').to_base_units().units == units.radian
     assert units('degrees_east') == units('degrees')
     assert units('degrees_east').to_base_units().units == units.radian
+    assert units('dBZ') == units('dB')
```

Comment:
Let's actually test that we have this correct in terms of log->linear:
```suggestion
    assert_almost_equal(0 * units.dBz, 1 * units('mm^6/m^3'))
```